setup.py

```diff
@@ -39,3 +39,27 @@
     },
     package_data={'beastling': ['data/*']},
 )
+
+setup(
```

Comment:
Yes! I just thought this would give a cleaner history until I'm sure it goes into Beastling or somewhere different.